benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -316,24 +316,22 @@ msgstr ""
 "To function properly, this website requires a browser that supports "
 "JavaScript. Please enable JavaScript for this website and"
 
-msgid "eligibility.pages.index.login_gov.label"
-msgstr "I am 65 years of age or older"
-
-msgid "eligibility.pages.index.login_gov.description"
-msgstr ""
-"You must be 65 years or older. This benefit does not expire, but you may "
-"need to renew. Using this benefit means your new transit fare is half of the "
-"standard fare."
-
-msgid "eligibility.pages.index.mst_cc.label"
+msgid "eligibility.pages.index.courtesy_card.label"
 msgstr "I have an MST Courtesy Card"
 
-msgid "eligibility.pages.index.mst_cc.description"
+msgid "eligibility.pages.index.courtesy_card.description"
```

Comment:
Same, I think `mst` makes sense here.

Reply:
Added `mst` to it so that the msgid is `eligibility.pages.index.mst_courtesy_card.description` in [95d85ca](https://github.com/cal-itp/benefits/pull/1513/commits/95d85cadc21a692d83d09e425032d3f9b4cfcb15). Is that the msgid you were picturing?